.github/workflows/JavaWithGradleTest.yml

```diff
@@ -24,3 +22,16 @@ jobs:
       run: chmod +x gradlew
     - name: Build with Gradle
       run: ./gradlew build
+
+
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run deploy to Github pages
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: ./Github Page
```

Comment:
I dont think this works/worked.
Try using a directory/path without a space, for example:
`publish_dir: ./GithubPage`